packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md

```diff
@@ -3,26 +3,53 @@ id: Writing design guidelines for PatternFly
+Design guidelines walk through building design patterns with PatternFly. This page focuses on how to write clear and consistent design documentation. To learn how to contribute design guidelines, see our [contribution guide](https://github.com/patternfly/patternfly-org/wiki/Contributing-to-patternfly-org-for-designers).
 
-## Content architecture
+## Design guidelines template
+When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must be under one of the following headings.
 
-Use headings, hierarchy, formatting, and lists to organize your design guidelines into digestible sections.
+### Elements (H2)
+Introduce the component and list the primary parts that make up its structure. This includes specific icons, headers, fields, symbols, or anything else included in the component.
+Example:
 
-### Headings
+<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
 
-Use headings to divide your content into topic-specific sections for easy navigation. Effective headings are:
+### Usage (H2)
+Explain how the component is used, when to use it, and why it should be used. Include explanations of problems it could help solve and address its advantages over similar components. Include images for visual examples.
 
-- Structured around users' main goals.
-- Front-loaded with descriptors for easy scanning.
-- Attentive to parallel structure.
-- Written in sentence case.
+Additional sub-sections to include: 
+#### When to use
+#### When not to use
+#### When to use X vs. Y
+#### Examples of incorrect usage
+
+
+### Behavior (H2)
+Explain how the component functions and break down how its elements interact with each other to achieve its purpose.
+
+### Variations (H2)
+Showcase different forms of your component and include explanations indicating when to use each one. 
+
+Additional sub-sections to include:
+#### When to use X variation vs. when to use Y variation
+
+### Spacing (H2)
+Introduce spacing requirements for components and their content. For more information regarding proper spacing, refer to our [spacing guidelines.](https://www.patternfly.org/v4/guidelines/spacers/)
+
+### Content considerations (H2)
+Showcase the different content that could be included within your component. Here is where you can also give additional tips and tricks the user may want to know. Content guidance includes:
+
+- Editorial guidelines for labeling and message text.
+- Length restrictions and what to do when text overflows.
+- Localization considerations.
+
+### Accessibility (H2)
+Each component should list the following sentence:
+
+For information regarding accessibility, visit the [component name] accessibility tab [linked].
```

Comment:
This is very confusing - I don't think i'd understand what to link looking at this. Worth adding a sentence to specify which part of the text should be linked in all this